starlette/responses.py

```diff
@@ -153,12 +153,23 @@ class PlainTextResponse(Response):
 class JSONResponse(Response):
     media_type = "application/json"
 
+    def __init__(
+        self, 
+        content: typing.Any = None, 
+        status_code: int = 200, 
+        headers: dict = None, 
+        media_type: str = None, 
+        background: BackgroundTask = None, 
+        indent: int = None) -> None:
```

Comment:
As you can see [here](https://github.com/encode/starlette/runs/3392828918) `black` will format this differently.
You can run `scripts/lint` locally.

Reply:
the newest commit should fix the formatting issue